src/smart-components/role/add-role-permissions/schema.js

```diff
@@ -52,11 +52,28 @@ export const schemaBuilder = (container) => {
                 component: 'cost-resources',
                 name: 'cost-resources',
                 validate: [
-                  (value = []) => (value.every((p) => p.resources.length > 0) ? undefined : intl.formatMessage(messages.assignAtLeastOneResource)),
+                  (value = []) =>
+                    value.every((p) => p.resources && p.resources.length > 0) ? undefined : intl.formatMessage(messages.assignAtLeastOneResource),
                 ],
               },
             ],
           },
+          {
+            name: 'inventory-groups-role',
```

Comment:
we are probably missing some condition that would reference this step, it does not seem to be visible